debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/MultiThreadIncrementalSnapshotIT.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.entry;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+
+/**
+ * Test to verify multi-thread incremental snapshotting for MongoDB.
+ *
+ * @author Yue Wang
+ */
+public class MultiThreadIncrementalSnapshotIT extends IncrementalSnapshotIT {
```

Comment:
You should override `io.debezium.connector.mongodb.IncrementalSnapshotIT.config()` here and set `MongoDbConnectorConfig.SNAPSHOT_MAX_THREADS`.
Ideally the inherited tests will just pass with it.

Reply:
done